hotel-reservation-system/src/components/Filters/DateFilter.js

```diff
@@ -101,7 +97,7 @@ export default function DateFilter({
         <Typography variant="h6">Check out date</Typography>
         <KeyboardDatePicker
           disableToolbar
-          minDate={new Date(moment(checkIn).add(2, 'days')._d)}
+          minDate={new Date(moment(checkIn).add(2, "days")._d)}
```

Comment:
Do you really need to wrap the moment entity into a new Date? Can't you just use moment(checkIn).add(2, 'days') or moment(checkIn).add(2, 'days')._d?

Reply:
changed to moment(checkIn).add(2, 'days')._d